src/coefficients/stdfunction_coefficient.cpp

```diff
@@ -30,6 +30,21 @@ void StdFunctionVectorCoefficient::Eval(mfem::Vector &V, mfem::ElementTransforma
   m_func(transip, V);
 }
 
+void MakeTrueEssList(mfem::ParFiniteElementSpace &pfes, mfem::VectorCoefficient &c, mfem::Array<int> &ess_tdof_list)
```

Comment:
@jamiebramwell @white238 I actually think I like the loop here better than the one in `MakeEssList`. Do you think it would be better to update `MakeEssList` with the forloop here and then have `MakeTrueEssList` just remap the `GetLocalTDofNumber`? Just a thought.

Reply:
I wasn't sure if you needed it the other way for some reason. I'll update this.

Reply:
No. When I wrote the precursor code, I don't think these methods existed.